ui/src/components/FileUpload.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <uppy-dashboard v-if="uppy" :uppy="uppy" :props="uppyDashboardOptions" />
+  <loading-block v-else />
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator'
+import AwsS3Multipart from '@uppy/aws-s3-multipart'
+import Uppy, { UppyFile } from '@uppy/core'
+import { Dashboard as UppyDashboard } from '@uppy/vue'
+import { prepareHeaders } from '@/api'
+import LoadingBlock from './LoadingBlock.vue'
+
+import '@uppy/core/dist/style.css'
+import '@uppy/dashboard/dist/style.css'
+
+const apiURL = window.APP_CONFIG.apiCoreBase
+// Maybe provide this URL in the Hydra API at some point
+const uploadURL = `${apiURL}upload`
+
+export interface UploadedFile extends UppyFile {
+  uploadURL: string
+  s3Multipart: {
+    key: string
+  }
+}
+
+@Component({
+  components: { LoadingBlock, UppyDashboard },
+})
+export default class extends Vue {
+  @Prop() afterUpload: any
+  @Prop() fileMeta: any
+  @Prop({ default: true }) allowMultiple!: boolean
```

Comment:
Do we ever change the default value of this prop?

Reply:
Not yet, we're going to need it if we implement the "replace file" feature.